gui/packages/ubuntupro/test/pages/landscape/landscape_page_test.dart

```diff
@@ -360,3 +404,34 @@ Widget buildApp(
 
 const customConf = './test/testdata/landscape/custom.conf';
 const notFoundPath = './test/testdata/landscape/notfound.txt';
+const caCert = './test/testdata/certs/ca_cert.pem';
+const clientCert = './test/testdata/certs/client_cert.pem';
+
+class FakeFilePicker extends FilePicker {
+  /// Fake [FilePicker] that always returns the given `paths`.
+  FakeFilePicker(this.paths);
+
+  final List<String> paths;
+
+  @override
+  Future<FilePickerResult?> pickFiles({
+    String? dialogTitle,
+    String? initialDirectory,
+    FileType type = FileType.any,
+    List<String>? allowedExtensions,
+    Function(FilePickerStatus p1)? onFileLoading,
+    bool allowCompression = true,
+    int compressionQuality = 30,
+    bool allowMultiple = false,
+    bool withData = false,
+    bool withReadStream = false,
+    bool lockParentWindow = false,
+    bool readSequential = false,
+  }) {
+    return Future(
+      () async => FilePickerResult(
+        paths.map((p) => PlatformFile(name: p, path: p, size: 0)).toList(),
+      ),
+    );
+  }
```

Comment:
This can be simplified into:

```suggestion
  }) async {
    return FilePickerResult(
        paths.map((p) => PlatformFile(name: p, path: p, size: 0)).toList(),
    );
  }
```

Reply:
or even

```suggestion
  }) async =>
    FilePickerResult(
        paths.map((p) => PlatformFile(name: p, path: p, size: 0)).toList(),
    )
```

Reply:
The mere fact that you declare your function (or method) `async` makes the return type a `Future` even if you just return a value without actually performing any asynchronous step.

Reply:
That's what I get for trusting the auto stub generation, should be good now :)